### ISSUE TITLE
20100...

### ISSUE THREAD

**Issue by tifaxx93** [barronwaffles/dwc_network_server_emulator#305]
Hi !
I'm french and i've a problem ...
I make the dns but alway error 20100...
I play with my Ds Lite .
How can i fix it? :/
(Sorry for my bad english) 
Thanks !

**Comment by Rottweiler** [comment]
Read Wiki https://github.com/polaris-/dwc_network_server_emulator/wiki/Troubleshooting

20100: Connection to the Access Point succeeded, but connection to the Wi-Fi Connection servers couldn't be established. Either you forgot to set the DNS and are trying to connect to the now-defunct official servers, or the custom server is currently down for maintenance. May also be a result of not properly patching your game to remove the SSL checks.

tl;dr server is malfunctioning or your game didn't patch correctly
